src/style.css

```diff
@@ -1 +1,79 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf');
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.search-group {
+  display: flex;
+  flex-direction: column;
+  gap: 20px; /* Espaçamento de 20px entre os elementos filhos */
+  padding: 20px 8px;
+}
+
+.search__bar {
+  font-size: 16px;
+  box-sizing: border-box;
+  border: 1px solid rgba(225, 231, 237, 1);
+  width: 100%; /* Ajuste para largura total */
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. For example, 'font-family: Avenir, Arial, sans-serif;' is a good practice as you have done, but ensure that 'Arial' is not the only fallback font. Consider adding another generic family such as 'Helvetica' to improve cross-platform compatibility.